### PR TITLE
added business-list components

### PR DESCRIPTION
**Pull request by shaikmutherpasha**
add bootstrap css js and jquery locally in folders

### CODE REVIEW

File: business.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+h1{
+      border-bottom: 1px solid #dddddd;
+      margin-bottom: 25px;
+      padding-bottom: 20px;
+    }
```
Comment (smfhashmi) [inline business.css]: add extra line after ending the file

File: business-list.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Business Table</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.7/yeti/bootstrap.min.css">
```
Comment (smfhashmi) [inline business-list.html]: please make local files and fetch them

File: business-list.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                  <td>Column content</td>
+                </tr>
+          </tbody>
+      </table> 
+
+  </div>
+</div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```
Comment (smfhashmi) [inline business-list.html]: download the files from these links and add it in the project and give the local path to it

File: business.css (hunk 1 above)
Comment (shaikmutherpasha) [reply business.css]: edited

File: business-list.html (hunk 2 above)
Comment (shaikmutherpasha) [reply business-list.html]: ok done

File: business-list.html (hunk 3 above)
Comment (shaikmutherpasha) [reply business-list.html]: done

File: business.css (hunk 1 above)
Comment (smfhashmi) [reply business.css]: its not reflecting

File: business-list.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Business Table</title>
```
Comment (smfhashmi) [inline business-list.html:4]: where are the local files for jquery bootstrap

**Pull request merged**